docs/tutorial/security.md

```diff
@@ -44,41 +44,495 @@ effort are always very welcome.
+* [Do not enable experimental features](#do-not-enable-experimental-features)
+* [Do not use `blinkFeatures`](#do-not-use-blinkfeatures)
+* [WebViews: Do not use `allowpopups`](#do-not-use-allowpopups)
+* [WebViews: Verify the options and params of all `<webview>` tags](#verify-webview-options-before-creation)
+
+
+## Only Display Secure Content
+
+Any resources not included with your application should be loaded using a
+secure protocol like `HTTPS`. In other words, do not use insecure protocols
+like
+
+### Why?
+
+`HTTPS` has three main benefits:
+
+1) It authenticates the remote server, ensuring that the host is actually whom
+   it claims to be. When loading a resource from an `HTTPS` host, it prevents
+   an attacker from impersonating that host, thus ensuring that the computer
+   your app's users are connecting to is actually the host you wanted them to
+   connect to.
+2) It ensures data integrity, asserting that the data was not modified while in
+   transit between your application and the host.
+3) It encryps the traffic between your user and the destination host, making it
+   more difficult to eavesdropping on the information sent between your app and
+   the host.
+
+### How?
+
+```js
+// Bad
+browserWindow.loadURL('http://my-website.com')
+
+// Good
+browserWindow.loadURL('https://my-website.com')
+```
+
+```html
+<!-- Bad -->
+<script crossorigin src="http://cdn.com/react.js"></script>
+<link rel="stylesheet" href="http://cdn.com/style.css">
+
+<!-- Good -->
+<script crossorigin src="https://cdn.com/react.js"></script>
+<link rel="stylesheet" href="https://cdn.com/style.css">
+```
+
+
+## Disable Node Integration for Remote Content
+
+It is paramount that you disable Node integration in any renderer
+([`BrowserWindow`](browser-view), [`BrowserView`](browser-view), or
+[`WebView`](web-view)) that loads remote content. The goal of disabling Node
+integration is to limit the powers you grant to remote content, thus making it
+dramatically more difficult for an attacker to harm your users should they gain
+the ability to execute JavaScript on your website.
+
+Disabling Node integration does not mean that you cannot grant additional
+powers to the website you are loading. If you are opening a
+[`BrowserWindow`](browser-window) pointed at `https://my-website.com`, the
+goal is to give that website exactly the abilities it needs, but no more.
+
+### Why?
+
+A cross-site-scripting (XSS) attack becomes dramatically more dangerous if an
+attacker can jump out of the renderer process and execute code on the user's
+computer. Cross-site-scripting attacks are fairly common - and while an issue,
+their power is usually limited to messing with the website that they are
+executed on. However, in a renderer process with Node.js integration enabled,
+an XSS attack becomes a whole different class of attack: A so-called "Remote
+Code Execution" (RCE) attack. Disabling Node.js integration limits the power
```

Comment:
nit: "Node.js integration" and "Node integration" are used interchangeably throughout this document. Can there be only one? ⚔️